hack/build-release.sh

```diff
@@ -0,0 +1 @@
+#place holder to minimize patching of hack/release.sh from origin
```

Comment:
Is it executed in Jenkins? If so, I guess it should have a shebang to don't throw errors.